test/forms/bold-media.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:enk="http://enketo.org/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:oc="http://openclinica.org/xforms" xmlns:odk="http://www.opendatakit.org/xforms" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+	<h:head>
+		<h:title>01Form</h:title>
+		<model odk:xforms-version="1.0.0">
+			<instance>
+				<data id="Bold_issue" version="1">
+					<note/>
+					<meta>
+						<instanceID/>
+					</meta>
+				</data>
+			</instance>
+			<bind nodeset="/data/note" readonly="true()" type="string"/>
+			<bind jr:preload="uid" nodeset="/data/meta/instanceID" readonly="true()" type="string"/>
+			<instance id="_users" src="jr://file/users.xml"/>
```

Comment:
In continuation of my question above, this is key to the test because it means `mediaMap != null`  is true even though the media file is never used?

Reply:
I have now explicitly provided a mediaMap in the test. Thanks!